recipes-init/openrc/openrc_0.54.2.bb

```diff
@@ -14,10 +14,11 @@ S = "${WORKDIR}/git"
 
 inherit meson
 
-PACKAGECONFIG ??= "${@bb.utils.filter('DISTRO_FEATURES', 'audit pam selinux usrmerge', d)}"
+PACKAGECONFIG ??= "${@bb.utils.filter('DISTRO_FEATURES', 'audit capabilities pam selinux usrmerge', d)}"
```

Comment:
I don't think that capabilities is a DISTRO_FEATURE (https://docs.yoctoproject.org/ref-manual/features.html#ref-features-distro), but it probably should be a default.  You can just add it on either end of the `${@......}`.

Reply:
ok, moved from `DISTRO_FEATURES` to standard config `capabilities`, as this global feature, there are lots of variants even in poky: `cap`, `libcap`, `capabilities`.